src/HttpServerFactory.php

```diff
@@ -63,9 +64,14 @@ class HttpServerFactory
      */
     public function __invoke(ContainerInterface $container): SwooleHttpServer
     {
-        $config       = $container->get('config');
+        $config = $container->get('config');
+        Assert::isMap($config);
```

Comment:
One potential problem here: in some DIC implementations, the `config` service is an `ArrayObject` for... reasons? :| 

This may break, so perhaps we should say that `\assert(is_array($config) || $config instanceof \ArrayAccess)`

Reply:
Fixed - thanks!